packages/design-system/src/components/icon-button.tsx

```diff
@@ -0,0 +1,86 @@
+import { styled } from "../stitches.config";
+
+export const IconButton = styled("button", {
+  // reset styles
+  boxSizing: "border-box",
+  padding: 0,
+  appearance: "none",
+  backgroundColor: "transparent",
+  border: "1px solid transparent",
+  // center icon
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  // prevent shrinking inside flex box
+  flexShrink: 0,
+  // set size and shape
+  width: 28,
+  height: 28,
+  borderRadius: 3,
+
+  "&:focus-visible": {
+    outline: "2px solid $blue10",
+  },
+  "&:disabled": {
+    borderColor: "transparent",
+    pointerEvents: "none",
+  },
+
+  variants: {
+    variant: {
+      default: {
+        color: "$slate12",
+        "&:hover": {
+          backgroundColor: "$slate6",
+        },
+        "&:disabled": {
+          color: "$slate8",
+        },
+      },
+      preset: {
+        backgroundColor: "$slate6",
+        borderColor: "$slate8",
+        color: "$slate12",
+        "&:hover": {
+          backgroundColor: "$slate8",
+        },
+        "&:disabled": {
+          color: "$slate8",
+        },
+      },
+      set: {
+        backgroundColor: "$blue4",
+        borderColor: "$blue6",
+        color: "$blue11",
+        "&:hover": {
+          backgroundColor: "$blue6",
+        },
+        "&:disabled": {
+          color: "$blue6",
+        },
+      },
+      inherited: {
+        backgroundColor: "$orange4",
+        borderColor: "$orange6",
+        color: "$orange11",
+        "&:hover": {
+          backgroundColor: "$orange6",
+        },
+        "&:disabled": {
+          color: "$orange6",
+        },
+      },
+      active: {
+        backgroundColor: "$blue10",
+        color: "White",
```

Comment:
Will `$loContrast` work instead of white?

Reply:
if it doesn't just fui we will be switching to completely new semantic colors set to support light/dark mode soon, we are actively working on it and soon will have to go over all colors and set them

Reply:
I was trying to say that if `$loContrast` gives a good result, we should replace "white" with it. 

Do you think we should leave as "white" because we're going to refactor colors?

Reply:
I haven't looked what's correct in this case in figma

Reply:
Yeah, I just think we should not leave as "white", can use any variable